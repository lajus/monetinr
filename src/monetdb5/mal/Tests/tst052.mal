#dealing with wrd BATs
# -1 + 2^n

b:= bat.new(:oid,:wrd);

bat.append(b,1:wrd);
bat.append(b,127:wrd);
bat.append(b,32768:wrd);
bat.append(b,2147483647:wrd);

io.print(b);
