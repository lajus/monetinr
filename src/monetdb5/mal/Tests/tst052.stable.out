stdout of test 'tst052` in directory 'monetdb5/mal` itself:


# 08:54:36 >  
# 08:54:36 >   mserver5 "--config=/ufs/mk/monet5/Linux/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=4 --set "monet_mod_path=/ufs/mk/monet5//Linux/lib/MonetDB5:/ufs/mk/monet5//Linux/lib/MonetDB5/lib:/ufs/mk/monet5//Linux/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/mk/monet5//Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/monet5//Linux/var/MonetDB5/sql_logs"  --set mapi_open=true --set xrpc_open=true --set mapi_port=32545 --set xrpc_port=44251 --set monet_prompt= --trace  --dbname=mTests_src_mal  tst052.mal
# 08:54:36 >  

# MonetDB server v5.7.0, based on kernel v1.25.0
# Serving database 'mTests_src_mal'
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked using 4 threads
# Copyright (c) 1993-2008 CWI, all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://eir.ins.cwi.nl:32545/
##dealing with wrd BATs
## -1 + 2^n
#
#b:= bat.new(:int,:wrd);
#
#bat.insert(b,1,1:wrd);
#bat.insert(b,7,127:wrd);
#bat.insert(b,15,32768:wrd);
#bat.insert(b,31,2147483647:wrd);
#bat.insert(b,63,9223372036854775807:wrd);
#
#io.print(b);
#
function user.main():void;
#dealing with wrd BATs 
# -1 + 2^n 
    b := bat.new(:oid,:wrd);
    bat.append(b,1:wrd);
    bat.append(b,127:wrd);
    bat.append(b,32768:wrd);
    bat.append(b,2147483647:wrd);
    io.print(b);
end main;
#-------------------------#
# h	t			  # name
# void	wrd		  # type
#-------------------------#
[ 0@0,	  1		  ]
[ 1@0,	  127		  ]
[ 2@0,	  32768		  ]
[ 3@0,	  2147483647	  ]

# 08:54:36 >  
# 08:54:36 >  Done.
# 08:54:36 >  

